corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -17,29 +17,35 @@ def translate(t, lang, langs=None):
 
 
 @register.filter
-def trans(name, langs=None, include_lang=True, use_delim=True):
+def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False):
     langs = langs or ["default"]
     if include_lang:
         if use_delim:
-            suffix = lambda lang: ' [%s]' % lang
+            tag = lambda lang: ' [%s]' % lang
         else:
-            suffix = lambda lang: '''
+            tag = lambda lang: '''
                 <span class="btn btn-xs btn-info btn-langcode-preprocessed">%(lang)s</span>
             ''' % {"lang": html.escape(lang)}
     else:
-        suffix = lambda lang: ""
+        tag = lambda lang: ""
     for lang in langs:
         if lang in name and name[lang]:
-            return name[lang] + ("" if langs and lang == langs[0] else suffix(lang))
+            affix = ("" if langs and lang == langs[0] else tag(lang))
+            return affix + name[lang] if prefix else name[lang] + affix
         # ok, nothing yet... just return anything in name
     for lang, n in sorted(name.items()):
-        return n + suffix(lang)
+        affix = tag(lang)
+        return affix + n if prefix else n + affix
     return ""
 
 
 @register.filter
 def html_trans(name, langs=["default"]):
-    return mark_safe(html.strip_tags(trans(name, langs, use_delim=False)) or EMPTY_LABEL)
+    return mark_safe(trans(name, langs, use_delim=False) or EMPTY_LABEL)
```

Comment:
nope, actually that is intentional. I give the lang prefix a class so that looks like 
<img width="151" alt="screen shot 2016-12-22 at 4 06 26 pm" src="https://cloud.githubusercontent.com/assets/716573/21440372/b385b262-c860-11e6-967e-82a444bf8c39.png">

If I add the strip_tags, it removes that html